npm/pkg/dataplane/ipsets/ipsetmanager_test.go

```diff
@@ -1546,6 +1546,11 @@ func TestValidateIPSetMemberIP(t *testing.T) {
 			ipblock: "2345:0425:2CA1:0000:0000:0567:5673:23b5 nomatch",
 			want:    false,
 		},
+		{
```

Comment:
not sure if there's a test for valid /0 yet

Reply:
updated